event/include/FindableTrack.h

```diff
@@ -0,0 +1,69 @@
+#ifndef FINDABLE_TRACK_H
+#define FINDABLE_TRACK_H
+
+/*~~~~~~~~~~*/
+/*   ROOT   */
+/*~~~~~~~~~~*/
+#include "TObject.h" 
+
+class FindableTrack { 
```

Comment:
Shouldn't this derive from TObject?
I know that this is no longer required, but it does help when serializing an object.

Reply:
... also helps when playing around on the root command line....